{"name": "Тест перехода на страницу Историю заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:505:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:135:16", "trace": "self = <diplom_3.tests.test_profile.TestProfile object at 0x000001CF99C62390>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"a47489a6-5b00-41d4-b931-5ab1b74d3f00\")>\n\n    @allure.title('Тест перехода на страницу Историю заказов')\n    def test_order_history(self, driver):\n        profile_page = ProfilePage(driver)\n>       profile_page.click_order_history()\n\ntests\\test_profile.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\profile_page.py:12: in click_order_history\n    self.click(profile_locators.order_history_button)\npages\\base_page.py:45: in click\n    button = self.wait(5).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"a47489a6-5b00-41d4-b931-5ab1b74d3f00\")>, method = <function element_to_be_clickable.<locals>._predicate at 0x000001CF99CE74C0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:505:5\nE       element.find/</<@chrome://remote/content/marionette/element.sys.mjs:135:16\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Инициализация BasePage с драйвером", "status": "passed", "parameters": [{"name": "driver", "value": "<selenium.webdriver.firefox.webdriver.WebDriver (session=\"a47489a6-5b00-41d4-b931-5ab1b74d3f00\")>"}], "start": 1717764057292, "stop": 1717764057292}, {"name": "Кликаем по кнопке История заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:505:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:135:16\n\n", "trace": "  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\pages\\profile_page.py\", line 12, in click_order_history\n    self.click(profile_locators.order_history_button)\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\pages\\base_page.py\", line 45, in click\n    button = self.wait(5).until(EC.element_to_be_clickable(locator))\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Клик по элементу с локатором", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:187:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:505:5\nelement.find/</<@chrome://remote/content/marionette/element.sys.mjs:135:16\n\n", "trace": "  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\pages\\base_page.py\", line 45, in click\n    button = self.wait(5).until(EC.element_to_be_clickable(locator))\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\i-mal\\PycharmProjects\\diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Ожидание, секунд", "status": "passed", "parameters": [{"name": "time", "value": "5"}], "start": 1717764057292, "stop": 1717764057292}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1717764057292, "stop": 1717764062380}], "start": 1717764057292, "stop": 1717764062383}], "parameters": [{"name": "driver", "value": "'Browser FireFox'"}], "start": 1717764057292, "stop": 1717764062387, "uuid": "d2fdb106-69c7-4590-98f7-c820bc5afebc", "historyId": "901c2db0376f302c44355fd04d80ff53", "testCaseId": "14856f90bd50d6a2988bf40c9ea8f95a", "fullName": "tests.test_profile.TestProfile#test_order_history", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_profile"}, {"name": "subSuite", "value": "TestProfile"}, {"name": "host", "value": "Lenovo-1"}, {"name": "thread", "value": "16996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_profile"}]}